libensemble/tests/run-tests.sh

```diff
@@ -261,6 +263,10 @@ while getopts ":p:n:a:y:hcszurmlt" opt; do
       echo "Running with user supplied test list"
       export REG_TEST_LIST="$OPTARG"
       ;;
+    W)
+      echo "Warning level parameter: $OPTARG" >&2
+      PYTHON_FLAGS='-W error'
```

Comment:
Should this be:
``PYTHON_FLAGS="-W $OPTARG"`` ?

Otherwise it doesn't matter what the required argument is, it will be ``error``.

Reply:
Yes, I think you are right.